src/components/editor/markdown-renderer/markdown-it-plugins/linkify-extra.ts

```diff
@@ -0,0 +1,13 @@
+import MarkdownIt from 'markdown-it/lib'
+import linkify from 'markdown-it/lib/rules_core/linkify'
```

Comment:
we could simply use [llinkify-it](https://www.npmjs.com/package/linkify-it) and then use [@types/linkify-it](https://www.npmjs.com/package/@types/linkify-it) instead of our own type. Not really sure if that's necessary, but it's possible

Reply:
linkify-it is not the markdown plugin. And I don't want to reinvent the markdown plugin.